cmake/build_helpers.cmake

```diff
@@ -113,26 +113,6 @@ macro(ginkgo_modify_flags name)
     string(REPLACE "\"" "\\\"" ${name}_MODIFY "${${name}}")
 endmacro()
 
-# Extract the clang version from a clang executable path
```

Comment:
I guess this is superseded by `CMAKE_CXX_COMPILER_VERSION`?

Reply:
Maybe, but this function was never used.